_rules/element-lang-matches-default-language-off6ek.md

```diff
@@ -303,7 +303,7 @@ There is no [text inheriting its programmatic language][] from this `p` element
 
 #### Inapplicable Example 6
 
-The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This is not a [valid language tag][].
+The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This is not a [known primary language tag][].
```

Comment:
```suggestion
The `lang` [attribute value][] of this `p` element is an [iso 639.2][] three letters code. This has no [known primary language tag][].
```
Or maybe go fully "Its primary tag is `eng`, which is not a known primary tag" 🤔

Reply:
Not sure I like that. How about this:

```suggestion
The `lang` [attribute value][] of this `p` element has no [known primary language tag][] because the `eng`[iso 639.2][] three letters code does not exist in the [language subtag registry][].
```